bokehjs/src/lib/models/markers/marker.ts

```diff
@@ -25,10 +27,20 @@ export abstract class MarkerView extends XYGlyphView {
   visuals: Marker.Visuals
 
   /** @internal */
-  glglyph?: MarkerGLGlyph
+  glglyph_cls?: Class<MarkerGL>
```

Comment:
`glglyph_cls` is temporary but the point is that GL-enablement for markers happens explicitly in the one place where the markers are defined, not implicitly in multiple parts of the codebase.